arreglo/src/main/java/com/globant/repository/ItemRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.globant.repository;
+
+import com.globant.model.Item;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface ItemRepository extends CrudRepository<Item,Integer>{
+
+    List<Item> findByOrderId(Integer orderId);
```

Comment:
this is provided by default when you etend from JpaRepository insted of CrudRepository